src/v/storage/tests/storage_e2e_test.cc

```diff
@@ -691,7 +691,7 @@ FIXTURE_TEST(write_concurrently_with_gc, storage_test_fixture) {
     };
 
     auto append =
-      [log, &_sem, &as, batches_per_append, &last_append_offset]() mutable {
+      [log, &_sem, batches_per_append, &last_append_offset]() mutable {
           return ss::with_semaphore(
             _sem, 1, [log, batches_per_append, &last_append_offset]() mutable {
                 return ss::sleep(10ms).then(
```

Comment:
nit: this could be sleep_abortable, but for 10ms I'm not sure it's worth it.

Reply:
yeh, hard to say what the original intention was, but maybe at some point the sleep was much longer?